paper/paper.md

```diff
@@ -0,0 +1,233 @@
+TransitionsInTimeseries.jl, which offers many additional advantages.
+
+# TransitionsInTimeseries.jl
+
+## Accessibility
+
+TransitionsInTimeseries.jl is accesible to any scientist thanks to the convenience functions
+it provides to detect and predict transitions in timeseries with only a few lines of code. A
+frequent prediction technique relies on observing, prior to a transition, an increase of the
+variance and the AR1 regression coefficient of the detrended time series, which is a
+consequence of Critical Slowing Down (CSD, [@scheffer_early-warning_2009]) and is here
+measured by the Kendall's $\tau$ coefficient. To assess whether this increase is significant,
+one can perform a statistical test, for instance by performing the same computations on 1,000
+surrogates of the original timeseries. The increase in variance and AR1 coefficient can be
+considered significant if the original timeseries classifies in the uppermost 5% of the
+surrogates, corresponding to a p-value $p<0.05$. All these steps can be performed, along with a minimal
+visualisation of the results within a few lines only:
+
+```julia
+# Loading and preprocessing the data needs to be done by the user
+time, data = load_data()
+
+# Choose the indicators and how to measure their change over time
+indicators = (var, ar1_whitenoise)
+change_metrics = (kendalltau, kendalltau)
+
+# Configuration with adequate parameters of the sliding window over a segment
+config = SegmentedWindowConfig(indicators, change_metrics, [t[1]], [t[end]];
+    width_ind = length(residual) ÷ 2, whichtime = last, min_width_cha = 100)
+
+# Compute the metrics over sliding windows and their significance
+results = estimate_indicator_changes(config, data, time)
+signif = SurrogatesSignificance(n = 1000, tail = :right, rng = Xoshiro(1995))
+flags = significant_transitions(results, signif)
+
+# A minimal visualisation of the results
+fig = plot_changes_significance(results, signif)
+```
+
+We apply this code to data generated by a Ricker model presenting an abrupt transition
+at $t = 860$, which is used in the first tutorial of `ewstools` [@bury_ewstools_2023],
+the most recent software with functionalities that are similar to TransitionsInTimeseries.jl.
+The results are shown in [Fig. 1](@figure1) and display, as expected from CSD theory, an
+increase in both variance and AR1 coefficient, which is exactly the same as computed
+by `ewstools`. However, calling `signif.pvalues` shows that the increase in variance is not
+significant ($p = 0.284$), whereas the increase in AR1 coefficient is ($p = 0.001$).
+
+![Output of plotting function in usage example.\label{fig: fig1}](figures/figure1.png)
+
+We believe that a concise and unambiguous code will greatly reduce the programming effort of
+many researchers and ease the code reviewing process. Finally, the code documentation
+provides additional examples, showcasing that the simplicity of the code also applies to
+real-world applications.
+
+## Performance
```

Comment:
I think we can shorten this section: just say that every step has been explicitly optimized and parallelized when possible (e.g., when computing timeseries surrogates for testing for significance). Explicitly refer to the performance comparison later on.